libglusterfs/src/inode.c

```diff
@@ -524,10 +524,8 @@ __inode_unref(inode_t *inode, bool clear)
     --inode->ref;
 
     index = __inode_get_xl_index(inode, this);
-    if (index >= 0) {
-        inode->_ctx[index].xl_key = this;
+    if (index >= 0)
```

Comment:
nit: our coding standard requires to always use `{}` even if the block contains a single line (https://github.com/gluster/glusterfs/blob/devel/doc/developer-guide/coding-standard.md#always-use-braces)

Reply:
Done.